### PR TITLE
documentation: ZENKO-1574_move_tox.ini_out_of_top_dir

### PR DESCRIPTION
**Pull request by wabernat**
This PR rationalizes the documentation structure some, and moves the tox.ini file out of the top-level Zenko directory.

It fixes ZENKO-1574

**Special notes for your reviewers**:

Moved tox.ini out of Zenko top-level directory down into docs, where it belongs

Changed Dockerfile, Makerfile

Changed tox.ini to support default HTML for all "books", latexpdf for individual
books (ops, inst, & ref)

Moved source files into "docsource" directory. This enables tox to capture
all three subdirectories.

RELENG: This is kludgey, but it works. There are probably some needlessly replicated files (Dockerfile, Makerfile, requirements.txt, etc.). Feel free to make it better!

### CODE REVIEW

File: eve/main.yml [hunk 1]
```diff
@@ -170,6 +170,7 @@ stages:
           # Building the latex twice to catch up the reference
           tox --workdir /tmp/tox -e docs -- latexpdf BUILDDIR="%(prop:builddir)s/build/artifacts"
           tox --workdir /tmp/tox -e docs -- latexpdf BUILDDIR="%(prop:builddir)s/build/artifacts"
+        workdir: docs/
```
Comment (ssalaues) [inline eve/main.yml]: I was mistaken, this actually needs to be `workdir: build/docs`. I just tried it and was able to build in CI.

File: eve/main.yml [hunk 2]
```diff
@@ -158,7 +158,7 @@ stages:
       path: eve/workers/doc-builder.yaml
       images:
         doc-builder:
```
Comment (tcarmet) [inline eve/main.yml:182]: here you can set the following `doc-builder: docs` as the context and the Dockerfile are in the same directory

File: docs/docsource/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,103 @@
+#
```
Comment (tcarmet) [inline docs/docsource/requirements.txt]: What's the difference between this file and https://github.com/scality/Zenko/blob/45b856ac80dbda1c9a42a31dc0c6f5da322f5883/docs/requirements.txt?

Are they the same? should we remove the one on the `docs` folder?

**Review by tcarmet — COMMENTED** [review]
I would recommend to checkup whats on the `docs` folder and ensure we're not duplicating things.

Other than that LGTM!  :)

**Comment by wabernat** [comment]
@bert-e approve

**Review by tcarmet — APPROVED** [review]

**Review by ssalaues — APPROVED** [review]

**Comment by ssalaues** [comment]
PR merged but was accidentally updated. There is no diff to merge and will close this PR.

**Pull request closed without merging**